api/api_sources/schema-migration-sql/MajorCitySchema/MajorCitySchema.sql

```diff
@@ -9,11 +9,11 @@ CREATE TABLE invasivesbc.major_city (
     closest_water_body character varying(1024),
     distance numeric,
     major_city_id integer NOT NULL,
-    created_at timestamp without time zone,
-    updated_at timestamp without time zone,
+    created_at timestamp without time zone default now()::timestamp,
+    updated_at timestamp without time zone default now()::timestamp,
     created_by_user_id character varying(100),
     updated_by_user_id character varying(100),
-    active_ind boolean
+    active boolean
```

Comment:
Why are we changing the name of this?

Reply:
I originally changed this because the table in Prod has it as `active`.  I assumed this needs to match it.

Reply:
I've taken the major cities stuff out of this PR for now anyways, as I was mainly just testing the migration capabilities. Dont need to over complicate this PR